proto/astria/sequencer/v1alpha1/transaction.proto

```diff
@@ -75,3 +76,18 @@ message MintAction {
   bytes to = 1;
   astria.primitive.v1.Uint128 amount = 2;
 }
+
+message Ics20Withdrawal {
+  astria.primitive.v1.Uint128 amount = 1;
+  string denom = 2;
+  string destination_chain_address = 3;
+  bytes return_address = 4;
```

Comment:
Our address type? 20 bytes?

Reply:
yeah it is, in the rust `Ics20Withdrawal` type this is an `Address`